autopts/ptsprojects/zephyr/gmcs.py

```diff
@@ -0,0 +1,104 @@
+#
+# auto-pts - The Bluetooth PTS Automation Framework
+#
+# Copyright (c) 2023, Codecoup.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms and conditions of the GNU General Public License,
+# version 2, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+
+"""GMCS test cases"""
+
+from autopts.pybtp import btp
+from autopts.client import get_unique_name
+from autopts.ptsprojects.stack import get_stack
+from autopts.ptsprojects.testcase import TestFunc
+from autopts.ptsprojects.zephyr.gmcs_wid import gmcs_wid_hdl
+from autopts.ptsprojects.zephyr.ztestcase import ZTestCase
+from autopts.pybtp.types import IOCap, Addr
+from queue import Queue
+
+queue = Queue()
+
+
+def set_addr(addr):
+    queue.put(addr)
```

Comment:
This is used in test cases with multiple lower testers to give a signal to LT2 that it can continue its pre_init. If pre_init of LT1 fails, LT2 hangs forever at `queue.get()` and because it is not interruptible under Windows, it could hang bot in our weekly run. I don't see any test case with LT2 in MSC.TS though, so you can just remove this.
```suggestion
from autopts.utils import ResultWithFlag

iut_addr = ResultWithFlag()

def set_addr(addr):
    iut_addr.set(addr)
```